### PR TITLE
[New] Add frappe-gantt@0.0.7 w/ npm auto-update

### LINKED ISSUE #12440: [Request] Add frappe-gantt
**Library name:** frappe-gantt
**Library description:** A simple, modern, interactive gantt library for the web
**Git repository url:** https://github.com/frappe/gantt
**npm package name or url** https://www.npmjs.com/package/frappe-gantt

### PR DESCRIPTION
**Pull request by Billy4195**
close #12440, cc @netchampfaris.

Pull request for issue: #12440
Related issue(s): 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/frappe/gantt
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/frappe-gantt
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 31
   - Count of stars: 571
   - Count of forks: 100
 * NPM download stats (optional):
   - Downloads in the last day: 21
   - Downloads in the last week: 337
   - Downloads in the last month: 718

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/frappe-gantt/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "type": "git",
+    "url": "https://github.com/frappe/gantt.git"
+  },
+  "keywords": [
+    "gantt",
+    "svg",
+    "simple gantt",
+    "project timeline",
+    "interactive gantt",
+    "project management"
```
Comment (extend1994) [inline ajax/libs/frappe-gantt/package.json]: Let's also add keywords shown in its git repo. (the block upper than commits, branches, releases, etc)

**Comment by Billy4195** [comment]
@extend1994 Finished!

**Comment by sufuf3** [comment]
@Billy4195 Would you please choose @extend1994 to request a review?
Here is a reference: https://help.github.com/articles/requesting-a-pull-request-review/
After you finish the PR, you could choose the reviewer to request a review this PR.
Many thanks

**Comment by Billy4195** [comment]
@sufuf3 OK, I got it. Thanks for reminding me.

**Pull request merged**